pilot/pkg/networking/plugin/authn/authentication.go

```diff
@@ -275,8 +275,12 @@ func buildSidecarListenerTLSContext(authenticationPolicy *authn.Policy) *auth.Do
 // OnOutboundListener is called whenever a new outbound listener is added to the LDS output for a given service
 // Can be used to add additional filters on the outbound path
 func (Plugin) OnOutboundListener(in *plugin.InputParams, mutable *plugin.MutableObjects) error {
-	// TODO: implementation
-	return nil
+	if in.Node.Type != model.Ingress {
```

Comment:
Actually, I think model.Ingress is the (old v1) ingress, and model.Router is the new ingress-gateway (that's what I interpret from istio.yaml, + @rshriram to confirm).  So this should be model.Router instead (or both)

Reply:
just mentioned in another comment, integration test shows that node.Type for istio-ingressgateway is 'router' now.